<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторна робота 3</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,5,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Дані</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leData"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>індекс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leIndex"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Список</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbAddAfter">
         <property name="text">
          <string>Додати після</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbAddBefore">
         <property name="text">
          <string>Додати перед</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDelete">
         <property name="text">
          <string>Видалити</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbFind">
         <property name="text">
          <string>Знайти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbGenerate">
         <property name="text">
          <string>Згенерувати</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbGiveTickets">
         <property name="text">
          <string>Роздати білети</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblStudents">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Студенти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbStudents">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblTickets">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Номери білетів</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbTicket">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lblResult">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Результат</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbResult">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="leResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
